src/Integration.UnitTests/Binding/BindingWorkflowTests.cs

```diff
@@ -200,32 +200,6 @@ public void BindingWorkflow_InstallPackages_FailureOnOneProject_Continues()
         }
 
         [TestMethod]
-        public void BindingWorkflow_PersistBinding()
```

Comment:
Moved this part to be included part of the solution binding operation in order to get the same file checkout experience for this info.